app/models/journal.rb

```diff
@@ -1,4 +1,20 @@
 class Journal < ApplicationRecord
   belongs_to :user
   belongs_to :journalable, polymorphic: true, required: false
+
+  # Categories
+  TELEPHONE = 'telephone'.freeze
+  CONVERSATION = 'conversation'.freeze
+  EMAIL = 'email'.freeze
+  FEEDBACK = 'feedback'.freeze
+  FILE = 'file'.freeze
+
+  CATEGORIES = [TELEPHONE, CONVERSATION, EMAIL, FEEDBACK, FILE].freeze
```

Comment:
I think in this case a:
```ruby
CATEGORIES = ['telephone',  'conversation',  'email', 'feedback', 'file'].freeze
```

would doo. Or do you need to access Journal::TELEPHONE ?